### PR TITLE
feat: Go profiling

### PR DESCRIPTION
**Pull request by vaind**
## Description of changes

Adds relevant pages to the new Go Profiling alfa support introduced in https://github.com/getsentry/sentry-go/pull/626

I've also adapted the `profiling-onboarding` wizard but don't know how to check that locally...

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by tonyo** [comment]
@lizokm how do we test changes to `wizard/` files these days?

**Review by aldy505 — APPROVED** [review]

**Review by vivianyentran — COMMENTED** [review]
Thanks for making these updates!

**Review by vivianyentran — APPROVED** [review]

**Comment by shanamatthews** [comment]
@tonyo - it's a bit of a process.

In your docs repo locally:
1. Create and run a production build of docs with your changes: 
  - `yarn build`
  - `yarn start`
 2. Make sure your changes show up at http://localhost:3000/_platforms/index.json
 
 In your sentry repo locally:
 3. Set `INTEGRATION_DOCS_URL="http://localhost:3000/_platforms/"` in your .env file
 4. Run sentry locally

You should see the changes in the wizard when you create a new project

File: src/platforms/common/profiling/index.mdx [hunk 1]
```diff
@@ -29,6 +29,16 @@ notSupported:
 description: "Learn how to enable profiling in your app if it is not already set up."
 ---
 
+<PlatformSection supported={["go"]}>
+
+<Note>
+
+Go Profiling is currently in alpha. Alpha features are still in-progress and may have bugs. We recognize the irony.
```
Comment (shanamatthews) [inline src/platforms/common/profiling/index.mdx:36]:
```suggestion
Go Profiling is currently in alpha. Alpha features are still in progress and may have bugs. We recognize the irony.
```

**Review by shanamatthews — APPROVED** [review]

**Comment by tonyo** [comment]
Tested the wizard changes locally, works 👍

**Pull request merged**